README.md

```diff
@@ -11,10 +13,24 @@
 - 📝 Line-specific comments on code changes
 - 🤖 Auto-fix suggestions with new PRs
 - 🎯 Project-specific guidelines
+- 🚫 Skip review functionality with special PR titles
 
 ## 🛠️ How to Use
 
-### 1. Set up Secrets
+### 1. Skip Code Review (Optional)
+To skip the code review for specific PRs, include any of these patterns in your PR title:
+```
+no-review: Update documentation
+skip review: Quick fix
+skip-code-review: Emergency patch
+otterai-skip: Configuration update
+otter skip: Minor changes
+otter-restricted: Sensitive update
+```
+
+The review will be skipped and Dr. OtterAI will leave a comment notifying the PR author.
```

Comment:
📢 Good to mention that Dr. OtterAI will notify the PR author when a review is skipped. This ensures transparency. 🔔